docs/plugins/form.md

```diff
@@ -176,15 +178,16 @@ The state contains information about the new novel name and its authors. Let's c
           <input formControlName="name" />
         </div>
       </div>
-
       <button type="submit">Create</button>
     </form>
   `
 })
-export class NewNovelComponent {
+export class NewNovelComponent implements OnInit  {
   newNovelForm: FormGroup;
   
-  constructor(private fb: FormBuilder) {
+  constructor(private fb: FormBuilder) {}
+
+  ngOnInit() {
```

Comment:
Why it's moved to `ngOnInit`?

Reply:
@arturovt I initialize all forms in my projects in ngOnInit, because all input parameters can only be accessed in this hook

Reply:
There are no input parameters. Please, revert it back to constructor.